src/coreclr/md/datablob.h

```diff
@@ -49,7 +49,7 @@ class DataBlob
     // Creates memory block copy.
     inline DataBlob(
         const DataBlob &source);
-    // Initializes memory block to empty data. The object could be already initialzied.
+    // Initializes memory block to empty data. The object could be already initializied.
```

Comment:
This (and other cases of it below) are still spelled incorrectly.
```suggestion
    // Initializes memory block to empty data. The object could be already initialized.
```